.github/workflows/activity_ci_cd.yml

```diff
@@ -4,11 +4,14 @@ on:
   push:
     branches: 
       - dev
-      - master
+      - staging
```

Comment:
@Kimaiyo077 do we need to adjust this when merging to master?

Reply:
No @andrewtpham , Only staging and dev should deploy on push